src/app/commands/tip/Tip.ts

```diff
@@ -0,0 +1,41 @@
+import { SlashCommand, CommandContext, CommandOptionType, SlashCreator } from 'slash-create';
```

Comment:
Tipping won't be active on DEGEN; it will be active on Mad Hatter. Please remove this file from this PR and use it in prep for tipping on Mad Matter:
https://github.com/BanklessDAO/degen-bankless-dao

Reply:
- removed from `DEGEN` repo. 🚮 ✅ 
- added new pr to [degen-bankless-dao](https://github.com/BanklessDAO/degen-bankless-dao/pull/57). ✨